packages/0xsequence/package.json

```diff
@@ -46,6 +46,7 @@
   "devDependencies": {
     "@babel/plugin-transform-runtime": "^7.12.1",
     "babel-loader": "^8.2.1",
+    "hardhat": "2.0.8",
```

Comment:
Do we need this here as we do define it in /package.json, and then all pkgs share node_modules. It's not a big deal if we have it here too, but just wondering if it wasn't working for you previously? or why else to add it?

Reply:
It will work if other packages also use hardhat as a dependency, but technically this package also does, and some tools give warnings if it's not properly defined